src/psyclone/tests/Master_test_errors.txt

```diff
@@ -0,0 +1,383 @@
+============================= test session starts ==============================
```

Comment:
Presumably this file has been accidentally added to the branch? Please delete.

Reply:
Deleted.